examples/index.html

```diff
@@ -22,9 +22,6 @@
 		require.config({
 			baseUrl: '..',
 			packages: [
-				{ name: 'tests', location: 'tests' },
-				{ name: 'dgrid', location: 'src' },
```

Comment:
do remember that 
a) relative paths are auto completed by most editors (including vim) out of the box 
b) obviously your examples are then AMD specific, ie: you might get end users who then try to do the same using webpack or just in node and wonder why they doesn't work.